src/uwtools/set_config.py

```diff
@@ -141,5 +147,20 @@ def create_config_obj(argv):
             out_object = config_obj
         out_object.dump_file(user_args.outfile)
 
+    ##begin show_format
+    # if --show_format, print a list of required configuration settings to stdout
+    if user_args.show_format:
+        if user_args.outfile is None:
+        # first ensure all required args are present
+            raise argparse.ArgumentError(user_args.outfile, \
+                "args: --show_format also requires -outfile for reference")
+
+        ref = out_object.__doc__
+        if outfile_type != infile_type and '\n' in ref:
+            # next, provide needed format if not performing a simple conversion
+            log.info(help(out_object.dump_file))
+        else:
+            log.info(r"No additional information is provided for this output file type")
+
```

Comment:
Why not just log whatever the doc string is? Can you not always call `help()` on a method? What if you just wanted to show the user the one-liner included with the out_object??

Reply:
I can call help(out_object) though its very long, unless there's a setting I'm not using correctly?

Reply:
I wasn't saying that you should call it on the object, just call it on the dump_file method, whether or not the __doc__ has a newline. It seems arbitrary to decide based on that.